src/core/ix86/iR3000A.cc

```diff
@@ -2353,23 +2359,27 @@ void X86DynaRecCPU::recSRAV() {
         m_iRegs[_Rd_].state = ST_UNK;
 
         gen.MOV32MtoR(PCSX::ix86::EAX, (uint32_t)&m_psxRegs.GPR.r[_Rt_]);
-        gen.MOV32ItoR(PCSX::ix86::ECX, m_iRegs[_Rs_].k & 0x1f);
+        gen.MOV32ItoR(PCSX::ix86::ECX, m_iRegs[_Rs_].k);
```

Comment:
I'd rather keep this one, as it's at jit time, and not execution time, and it would make it clearer for porting the JIT to other CPUs.